tpm2/include/memory_util.h

```diff
@@ -1,12 +1,12 @@
 /**
- * @file  tpm2_kmyth_misc.h
+ * @file  memory_util.h
  *
  * @brief Provides miscellaneous utility functions for Kmyth applications
```

Comment:
Should probably also change this comment from "miscellaneous utility functions" to "memory utility functions"